layout_padded/layout_padded.md

```diff
@@ -2529,17 +2531,20 @@ is `true`.  Otherwise, `false`.
 
 [1]{.pnum} *Returns:*
 
-   * [1.1]{.pnum} `submdspan_mapping_result{*this, 0}`, if `Extents::rank()==0` is `true`;
+   * [1.1]{.pnum} `submdspan_mapping_result{*this, 0}`, if _`rank_`_`==0` is `true`;
+   
+   * [1.2]{.pnum} otherwise, `submdspan_mapping_result{layout_left::mapping(sub_ext), offset}`, if `SubExtents::rank()==0` is `true`;
```

Comment:
Note to self to check that all the layout mappings' _`submdspan-mapping-impl`_ implementations have this optimization.

Reply:
They do : - )